instrumentation/elasticsearch/elasticsearch-api-client-8.0/javaagent/src/test/java/ElasticsearchClient8Test.java

```diff
@@ -0,0 +1,254 @@
+                            equalTo(SemanticAttributes.USER_AGENT_ORIGINAL, userAgent()),
+                            satisfies(
+                                SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH,
+                                AbstractLongAssert::isPositive))));
+  }
+
+  @Test
+  // ignore deprecation interface
+  public void elasticsearchIndex() throws IOException {
+    client.index(
+        r ->
+            r.id("test-id")
+                .index("test-index")
+                .document(new Person("person-name"))
+                .timeout(t -> t.time("1s")));
+
+    testing.waitAndAssertTraces(
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span ->
+                    span.hasName("index")
+                        .hasKind(SpanKind.CLIENT)
+                        .hasNoParent()
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(SemanticAttributes.DB_SYSTEM, "elasticsearch"),
+                            equalTo(SemanticAttributes.DB_OPERATION, "index"),
+                            equalTo(SemanticAttributes.HTTP_METHOD, "PUT"),
+                            equalTo(
+                                SemanticAttributes.HTTP_URL,
+                                httpHost.toURI() + "/test-index/_doc/test-id?timeout=1s"),
+                            equalTo(
+                                AttributeKey.stringKey("db.elasticsearch.path_parts.index"),
+                                "test-index"),
+                            equalTo(
+                                AttributeKey.stringKey("db.elasticsearch.path_parts.id"),
+                                "test-id")),
+                span ->
+                    span.hasName("PUT")
+                        .hasKind(SpanKind.CLIENT)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(SemanticAttributes.NET_PEER_NAME, httpHost.getHostName()),
+                            equalTo(SemanticAttributes.NET_PEER_PORT, httpHost.getPort()),
+                            equalTo(SemanticAttributes.HTTP_METHOD, "PUT"),
+                            equalTo(AttributeKey.stringKey("net.protocol.name"), "http"),
+                            equalTo(AttributeKey.stringKey("net.protocol.version"), "1.1"),
+                            equalTo(
+                                SemanticAttributes.HTTP_URL,
+                                httpHost.toURI() + "/test-index/_doc/test-id?timeout=1s"),
+                            equalTo(SemanticAttributes.HTTP_STATUS_CODE, 201L),
+                            equalTo(SemanticAttributes.USER_AGENT_ORIGINAL, userAgent()),
+                            satisfies(
+                                SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH,
+                                AbstractLongAssert::isPositive))));
+  }
+
+  @Test
+  // ignore deprecation interface
```

Comment:
I wasn't able to figure out what this comment means

Reply:
removed